bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/VersionToken.scala

```diff
@@ -0,0 +1,73 @@
+package edu.gemini.spModel.core
+
+import scalaz._
+import Scalaz._
+
+/**
+ * Simple integer version token (e.g., 1.2.3) used to distinguish VO Table
+ * versions.
+ */
+sealed abstract case class VersionToken private (toNel: NonEmptyList[Int]) {
```

Comment:
Perhaps this is overkill.  I just needed a way to specify a version number like "1.2.3".